string name
string bond_id
connection_pub_sub[] subscriptions
connection_pub_sub[] publications
